images/packages/binaries/gcc/werf.inc.yaml

```
@@ -0,0 +1,98 @@
+---
+image: {{ $.ImageType }}/{{ $.ImageName }}
+final: false
+fromImage: builder/scratch
+import:
+- image: {{ $.ImageType }}/{{ $.ImageName }}-builder
+  add: /out
+  to: /gcc
+  before: setup
+
+---
+{{- $version := "releases/gcc-14.2.0" }}
```

Use this to get the version from a file `component_versions/version_map.yml`
```suggestion
{{- $version := get $.Package $.ImageName }}
```